docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -0,0 +1,212 @@
+# Prebuilt workspaces
+
+Prebuilt workspaces allow template administrators to improve the developer experience by reducing workspace
+creation time with an automatically maintained pool of ready-to-use workspaces for specific parameter presets.
+
+The template administrator configures a template to provision prebuilt workspaces in the background, and then when a developer creates
+a new workspace with matching parameters, Coder assigns them an existing prebuilt instance.
+Prebuilt workspaces significantly reduce wait times, especially for templates with complex provisioning or lengthy startup procedures.
+
+Prebuilt workspaces are:
+
+- Created and maintained automatically by Coder to match your specified preset configurations.
+- Claimed transparently when developers create workspaces.
+- Monitored and replaced automatically to maintain your desired pool size.
+
+## Prerequisites
+
+- [**Premium license**](../../licensing/index.md)
+- **Compatible Terraform provider**: Use `coder/coder` Terraform provider `>= 2.3.0-pre2`. (**TODO: update with latest version**)
+- **Feature flag**: Enable the `workspace-prebuilds` [experiment](../../../reference/cli/server.md#--experiments).
+
+## Enable prebuilt workspaces for template presets
+
+In your template, add a `prebuilds` block within a `coder_workspace_preset` block to identify how many prebuilt
+instances your Coder deployment should maintain:
+
+   ```hcl
+   data "coder_workspace_preset" "goland" {
+     name = "GoLand: Large"
+     parameters = {
+       jetbrains_ide = "GO"
+       cpus          = 8
+       memory        = 16
+     }
+     prebuilds {
+       instances = 3  # Number of prebuilt workspaces to maintain
+     }
+   }
+   ```
+
+After you create or update the template, Coder automatically provisions and maintains prebuilt workspaces through an
+internal reconciliation loop (similar to Kubernetes) to maintain the number of specified `instances`.
+
+Prebuilt workspaces are displayed in the list of workspaces on the Coder dashboard with the owner set to `prebuilds`.
+
+## Prebuilt workspace lifecycle
+
+Prebuilt workspaces follow a specific lifecycle from creation through eligibility to claiming.
+
+Expand each item in this list for more information about the stage:
+
+1. <details><summary>The prebuilt workspace is provisioned.</summary>
```

Comment:
I don't know how keen I am on this yet.

The goal was to summarize the lifecycle in a way that people can see easily and drill down into for more details if they want. I'm going to try another edit of the summaries to see if I can get it to work, and if it doesn't, I'll revert to separate headings